config/routes.rb

```diff
@@ -1,8 +1,8 @@
 Rails.application.routes.draw do
   root "groups#index"
-  resources :memberships
+  resources :memberships, only: [:new, :create, :destroy]
   devise_for :users
-  resources :groups
-  resources :contacts
+  resources :groups, only: [:index, :show, :new, :create, :edit, :update, :destroy]
+  resources :contacts, only: [:index, :show, :new, :create, :edit, :update, :destroy]
```

Comment:
`:index, :show, :new, :create, :edit, :update, :destroy` are the defaults for the `resources` method. Since you aren't excluding you don't need to specify them in the `:only` key.